config/locales/en/main.yml

```diff
@@ -2266,18 +2270,18 @@ en:
 
   user_import:
     instructions_html: |-
-      Upload a spreadsheet with your users. Only XLSX-type Excel spreadsheets and CSV files are accepted.
-
-      A spreadsheet template is available in <a href="%{xlsx_url}">XLSX</a> and <a href="%{csv_url}">CSV</a> formats as a starting point.
+      Upload a spreadsheet with your users. A <a href="%{csv_url}">CSV template</a> is available as a starting point.
```

Comment:
Can we say "CSV file" instead of "spreadsheet"? The former is the more common way to refer to it I think. "spreadsheet" usually means .xlsx or .numbers or whatever. I don't think people think of CSV files as spreadsheets, do they?

Reply:
Agreed, the code comments are meant to be generic so that we don't need to change them again later, but user-facing docs should say CSV for clarity. Will do!